pai-management/doc/how-to-write-pai-configuration.md

```diff
@@ -131,25 +130,25 @@ kubernetes:
   dashboard-version: v1.8.3
 ```
 
-#### ```Some important value should set according to the specific cluster  ```
+#### ```User *must* set the following fields to bootstrap a cluster ```
 
 - ```cluster-dns```: Find the namesever address in  /etc/resolv.conf
-- ```load-balance-ip```: If in your cluster, there is only on k8s-master, you should set this field with the ip-address of your k8s-master. If there are many k8s-master, please refer to this [chapter](#k8s-high-availability-configuration).
-- ```service-cluster-ip-range```: In this field, you should specify an ip range which shouldn't overlap with your current host's ip in your cluster.
-
-
+- ```load-balance-ip```: If the cluster has only one k8s-master, please set this field with the ip-address of your k8s-master. If there are many k8s-master, please refer to [k8s high availability configuration](#k8s-high-availability-configuration).
 
 #### ```Some values could use the default value```
-- ```storage-backend```: If you are not familiar with etcd, please don't change it.
-- ```docker-registry```: The docker registry used in the k8s deployment. If you can access to gcr, we suggest to use gcr. After set this field with gcr.io/google_containers, your kubernetes component's image will pull from ```gcr.io/google_containers/hyperkube```
+- ```service-cluster-ip-range```: Please specify an ip range that does not overlap with the host network in the cluster. 169.254.0.0/16 link-local IPv4 address according to [RFC 3927](https://tools.ietf.org/html/rfc3927), which usually will not overlap with your cluster IP.
```

Comment:
I don't think we could say 169.254 will work. We never tested before.

Reply:
I tested it.

---
In reply to: [195305494](https://github.com/Microsoft/pai/pull/727#discussion_r195305494) [](ancestors = 195305494)

Reply:
I change it to 10.254.0.0/16 to be consistent to the default value in quick start deployment

---
In reply to: [195306192](https://github.com/Microsoft/pai/pull/727#discussion_r195306192) [](ancestors = 195306192,195305494)